.evergreen.yml

```diff
@@ -66,57 +66,82 @@ functions:
       aws_key: ${aws_key}
       aws_secret: ${aws_secret}
       local_files_include_filter:
-        ["parsley/storybook-static/*"]
+        ["parsley/storybook-static/*.html"]
       remote_file: parsley/${task_id}/storybook/
       bucket: mciuploads
       content_type: text/html
       permissions: public-read
       display_name: storybook_
+      preserve_path: true
 
-  # Ideally we can get rid of this command upon the completion of EVG-17334.
   attach-storybook-assets-js:
```

Comment:
What do you think of combining all the `attach-storybook-` functions into one function with multiple commands? I did this in https://github.com/evergreen-ci/spruce/pull/1640 and I think it is a little more readable.

Reply:
Good call done